src/main/java/domain/BettingAmount.java

```diff
@@ -0,0 +1,61 @@
+package domain;
+
+import java.util.Objects;
+
+public class BettingAmount {
+
+    private static final int MIN_AMOUNT = 10000;
+    private static final BettingAmount INITIAL = new BettingAmount(0);
+    private static final String MIN_BETTING_AMOUNT_ERROR_MESSAGE = "초기 배팅 금액은 10000원 이상이어야 합니다.";
```

Comment:
아래와 같이 표현하면 최소 베팅금액이 변경됐을 때, 상수만 변경하고 에러메시지는 수정하지 않는 휴먼에러를 방지할 수 있어요 😄 
```suggestion
    private static final String MIN_BETTING_AMOUNT_ERROR_MESSAGE = String.format("초기 배팅 금액은 %s원 이상이어야 합니다.", MIN_AMOUNT);
```